tests/test_pytest/test_plugin_schema/good/sectors/area_definition.yaml

```diff
@@ -0,0 +1,29 @@
+---
+interface: sectors
+family: area_definition
+name: CONUS
+docstring: |
+  CONUS sector AreaDefinition.
+metadata:
```

Comment:
I reall like this!

Reply:
I do too! I'm still not sure that I'm happy with `CCASSC` at this point, but this works well as long as we're using it!